roles/aodh/templates/etc/serverspec/aodh_spec.rb

```diff
@@ -33,6 +28,18 @@
   it { should be_owned_by 'aodh' }
 end
 
+{% if inventory_hostname in groups['controller'] %}
+processes = ['aodh-api',
+  'aodh-evaluator',
+  'aodh-listener',
+  'aodh-notifier']
+processes.each do |process|
+  describe process(process) do
+    its(:user) { should eq 'aodh' }
```

Comment:
what are we solving here?  I don't like the idea of testing a running process like this, I'd rather test the config file / upstart that sets the user.